cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
+there are actually a lot of decent resources online already on this, such as
+https://www.thenappygurus.com/types-of-cloth-nappies.html
+
+The whole of the [Nappy Gurus Help & Advice](https://www.thenappygurus.com/help.html)
+section is extremely well put together and very helfpul if you're starting out.
+
+I'm just leaving this diagram here as a reference point to discuss the
+choices we have made (see below).
+
+## Materials and Absorbency
+
+<img width="664" alt="image" src="https://user-images.githubusercontent.com/4185328/169095781-8c3c8f69-3ae5-4e04-ab39-780f198651a5.png">
+
+Cloth diapers hinge on both their speed of and capacity for absorbency.
+
+The only other thing to mention other than the table above is 'Charcoal Bamboo'.
+This is actually not bamboo at all, it's just a fancy name for microfibre and
```

Comment:
BTW: it _is_ bamboo that has undergone a specific industrial process/transformation ...
https://www.simplififabric.com/pages/bamboo-charcoal
"_The bamboo charcoal is created by heating bamboo at temperatures of 800 degrees and then the charcoal itself is processed and mixed in with fabrics as part of the growing field of nanotechnology._"

![image](https://user-images.githubusercontent.com/194400/173175209-3c742704-66b1-45a3-9090-e57d150f2d93.png)

Maybe consider revising? 💭

Reply:
Thanks, I'll clarify 👍 

Could you please confirm that if I add the suggested changes as below within the same comment it still works? 


```suggestion
These are bamboo only in name because there is very little if any bamboo in these.
They are a microfibre insert wrapped in an incredibly thin charcoal bamboo layer (and in quite a few cases it's arguably not a charcoal bamboo layer but just a charcoal coloured fleece layer, see https://youtu.be/d04Hy11xU58).
Ultimately this is a money-grabbing ploy from the companies that sell them - it's just a fancy name for microfibre and
```